homeassistant/components/overkiz/config_flow.py

```diff
@@ -144,9 +140,13 @@ async def async_step_zeroconf(
         gateway_id = properties["gateway_pin"]
 
         LOGGER.debug("ZeroConf discovery detected gateway %s", obfuscate_id(gateway_id))
+        return await self._process_discovery(gateway_id)
 
+    async def _process_discovery(self, gateway_id: str) -> FlowResult:
+        """Handle discovery of a gateway."""
         await self.async_set_unique_id(gateway_id)
         self._abort_if_unique_id_configured()
+        self.context["title_placeholders"] = {CONF_NAME: f"Gateway: {gateway_id}"}
```

Comment:
This isn't translatable since `Gateway` is inline here.

You can make it translatable by:

```suggestion
        self.context["title_placeholders"] = {'gateway_id': gateway_id}
```

Then add `"flow_title": "Gateway: {gateway_id}",` in `strings.json` (there are a few examples in other integrations if you `git grep`) and regenerate `en.json` with `python3 -m script.translations develop`

Reply:
thanks for the pointer. 

I tried to look for documentation on the context and elements that could be overwritten, but could not find any. Did I just miss it or is going through other integration the way to go?

Reply:
I think the docs could use some work.

Here is how `broadlink` does it
https://github.com/home-assistant/core/blob/dev/homeassistant/components/broadlink/strings.json#L3

Here is how `apple_tv` does it
https://github.com/home-assistant/core/blob/dev/homeassistant/components/apple_tv/strings.json#L3